packages/docs/content/not-in-sidebar/contact.mdx

```diff
@@ -3,10 +3,10 @@ title: Contact us
 intro: If you've got a question, idea, or suggestion, get in touch with the Design System team.
 ---
 
-import GithubIcon from '../../src/components/GithubIcon';
-import JiraIcon from '../../src/components/JiraIcon';
-import SlackIcon from '../../src/components/SlackIcon';
-import EmailIcon from '../../src/components/EmailIcon';
+import GithubIcon from '../../src/components/Icons/GithubIcon';
+import JiraIcon from '../../src/components/Icons/JiraIcon';
+import SlackIcon from '../../src/components/Icons/SlackIcon';
+import EmailIcon from '../../src/components/Icons/EmailIcon';
```

Comment:
I just realized that all my folders are capitalized even though the capitalization is only meaningful for `Layout` (because that folder's index points to a component of the same name). Should I instead lowercase them all and have the `Layout` component be referenced like `components/layout/Layout`?

Reply:
Yeah I think that's a good idea 👍🏻